core/payload/deploy_code.go

```diff
@@ -252,10 +287,13 @@ func CreateDeployCode(code []byte,
 	author []byte,
 	email []byte,
 	desc []byte) (*DeployCode, error) {
+	if vmType > 255 {
+		return nil, fmt.Errorf("wrong vm flags: %d", vmType)
+	}
```

Comment:
here is redundant. validateDeployCode already check

Reply:
this checks uint32 -> byte convension.